### PR TITLE
M3 focus item preview using iframe

### LINKED ISSUE #7179: Focus - blank screen in builder
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
The more details the better...

| Mautic version 2.15.0 | 
| PHP 7.0.33 | 
| Chrome browser on PC | 

### Steps to reproduce
1. When creating a new focus, and opening the builder, when I enter a domain name - e.g. thriveengine.nz - the screen is blank.
2. However, it seems to work on thriveengine.com, but strangely, not on other .com domains I've tried.

Here's a video showing what I see:
https://cl.ly/44e31aa6b399

### Log
I didn't find any errors, relating to focus, in /app/log/

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

### PR DESCRIPTION
**Pull request by hluchas**
| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | https://github.com/mautic/mautic-documentation/pull/72
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/7179
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
No website preview loaded. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to Channels / Focus items / Create new / click Builder
2. Enter page URL with protocol
3. Page will be not loaded

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat reproduction steps
3. Try to use different settings

### CODE REVIEW

**Review by kuzmany — DISMISSED** [review]
It works.
Just iframe is not scrollable, it's little anoying, but it's not bug.

File: plugins/MauticFocusBundle/Assets/css/focus.css [hunk 1]
```diff
@@ -16,16 +16,20 @@
...
   top: 0;
   bottom: 0;
   right: 0;
   left: 0;
   margin: auto;
+  z-index: 0;
 }
 .focus-builder #websiteScreenshot .screenshot-container {
-  overflow: hidden;
+  /*overflow: hidden;*/
```
Comment (kuzmany) [inline plugins/MauticFocusBundle/Assets/css/focus.css]: We need this?

Comment (hluchas) [reply plugins/MauticFocusBundle/Assets/css/focus.css]: Removed.

**Comment by hluchas** [comment]
> It works.
> Just iframe is not scrollable, it's little anoying, but it's not bug.

This was a part of internal specification. Keeping this as it is for now.
If ou want to change this, create issue ;-)

**Review by RCheesley — DISMISSED** [review]
Tested on Mautibox, worked fine once I found a domain that did not block the behaviour! We need to ensure that the docs are updated to explain why some sites will not be able to display a preview.

**Comment by RCheesley** [comment]
@hluchas edited to include a link to the docs PR I made, have requested your review on the PR.

**Review by MartinKiely — DISMISSED** [review]
Thanks for the updates and fixes to Focus Items. Having tested the changes on the issues experienced with Mautic v2.16.2 have been resolved.

**Comment by hluchas** [comment]
> Thanks for the updates and fixes to Focus Items. Having tested the changes on the issues experienced with Mautic v2.16.2 have been resolved.

You are welcome. Preview functionality was completely re

**Review by RCheesley — APPROVED** [review]
Good to go for 3.0.1

**Comment by dennisameling** [comment]
Merging as Travis tests have passed (https://travis-ci.org/github/mautic/mautic/builds/700150756), status is just not updated here.

**Pull request merged**